tutorials/notebooks/specutils-fit-line/specutils-fit-line1.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>Adding plotting functionality to <code>specutils</code>  is currently under discussion and you can read the details and contribute to the conversation on Github. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>done</p>